import-service/lambda/importFileParser.ts

```diff
@@ -0,0 +1,55 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { S3Client, PutObjectCommand } from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+
+const CORS_HEADERS = {
+  "Access-Control-Allow-Origin": "*",
+  "Access-Control-Allow-Credentials": true,
+};
+
+const s3Client = new S3Client({});
+
+export const handler = async (
+  event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> => {
+  console.log("Event:", JSON.stringify(event));
+
+  try {
+    const fileName = event.queryStringParameters?.name;
+
+    if (!fileName) {
+      return {
+        statusCode: 400,
+        headers: CORS_HEADERS,
+        body: JSON.stringify({ message: "Name parameter is required" }),
+      };
+    }
+
+    const command = new PutObjectCommand({
+      Bucket: process.env.BUCKET_NAME,
+      Key: `${process.env.UPLOADED_FOLDER}/${fileName}`,
+      ContentType: "text/csv",
+    });
+
+    const signedUrl = await getSignedUrl(s3Client, command, {
+      expiresIn: 3600,
+    });
+
+    console.log("Generated signed URL:", signedUrl);
+
+    return {
+      statusCode: 200,
+      headers: CORS_HEADERS,
+      body: JSON.stringify({
+        url: signedUrl,
+      }),
```

Comment:
It seems strange, coz by requirements you should in this function just handle read file by readable stream, parse it with csv parser and log each parsed record. I don't see even csvParcer usage. 
And
```
return {
      statusCode: 200,
      body: JSON.stringify({ message: 'Successfully processed files' })
};
```
will be enough in this case